### ISSUE TITLE
The workflow state is not immediately visible to the editor

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#1987]
I think the workflow state should be highlighly visible to the editor. In default Plone we even show with color-attention grabbing red, in case an item is private, so that the editor knows that the item needs to be published.

**Comment by tisto** [comment]
@tiberiuichim the workflow state is visible in the colored handle-bar in the edit sidebar.

**Comment by tiberiuichim** [comment]
haha... yeah, TIL. That is a very subtle cue, though.

**Comment by tisto** [comment]
Yeah. I agree. We also got feedback from a client that the colors we use for workflow states do not work for color blind people. Maybe something we should discuss with @albertcasado...

**Comment by albertcasado** [comment]
If you use a color bullet without label of state it will not be accessible.
If you use state with bullet the bullet is complementary so the user can know the status by reading the text. The color bullet is just fancy.
Since workflows has a lot of states I recommend you to report always state with text and the color bullet as hint for people that can see colors, no need to add shapes here

Enviat des del meu iPhone

**Comment by rotavio** [comment]
I've noticed that the handlebar doesn't turn to yellow (or orange) when it's being reviewed. It only gets red or blue. Is this by design?
